src/resolvers/platform/WindowsSecurity.js

```diff
@@ -94,14 +110,14 @@ export default {
         userApp = apps.find((app) => app.name === name)
       }
 
-      // app isn't installed - fail
-      if (!userApp) return { name, passing: false, reason: 'NOT_INSTALLED' }
-      // app is out of date - fail
+      // app isn't installed
+      if (!userApp) return { name, reason: 'NOT_INSTALLED' }
+      // app is out of date
       if (version && !semver.satisfies(userApp.version, version)) {
-        return { name, passing: false, reason: 'OUT_OF_DATE' }
+        return { name, version: userApp.version, reason: 'OUT_OF_DATE' }
```

Comment:
There is still a react UI bug where the suggested version is rendered as `undefined` when we're in this state. I note that here for completeness' sake.